src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -50,7 +50,7 @@ class FlowHelper {
 
     def random = new Random()
     def faker = new Faker()
-    def allowedVlans = 101..4095
+    static final IntRange ALLOWED_VLANS_ACCORDING_TO_IEEE_802_1_Q_PAGE_76 = 2..4094
```

Comment:
you see, now another question arises: why did you extend the range? What will be the implications of using VLANs 2..101. Why the range started at 101 previously?

Reply:
Why are you asking me? I guess, this question should (or even should have been, during PR review) asked to the person, who added this interval. 
Why it is 2..4094 is explained in IEEE 802.1Q standard, paragraph 9.6, page 76. 
Also, foreseeing your questions, I don't know, why engineers in IEEE decided to limit VLAN IDs to 12 bits only and why they decided to use exactly 0,1, and 4095 as reserved values. You should address these questions to IEEE committee.

Reply:
I am asking you because you changed the interval from 101..4095 to 101..4096 and then to 2..4094. And this is a legitimate interest why it is so and which interval is the correct one.
You said that the interval has been incorrect for years. I am asking whether it is correct now. Also I think that this is a problematic point which requires some attention. That is if this interval is correct then what about other places where it is starts from 101 or ends with 4095.
Clearly, this is a source of potential test instability or confusion. I propose to fix it properly.